dipy/stats/analysis.py

```diff
@@ -160,6 +160,10 @@ def gaussian_weights(bundle, n_points=100, return_mahalnobis=False,
     n_points : int, optional
         The number of points to resample to. *If the `bundle` is an array, this
         input is ignored*. Default: 100.
+    stat : callable, optional.
```

Comment:
Can you add the `return_mahalnobis` docstring also ?

Thank you

Reply:
Oh yeah - good catch! Implemented in https://github.com/dipy/dipy/pull/2683/commits/a20b113e2e5faf75bf985f0c5b5d7c67c079eba7